### ISSUE TITLE
Question about acclerate the simulation of executable environment

### ISSUE THREAD

**Issue by hxx1996** [Unity-Technologies/ml-agents#5654]
**Is your feature request related to a problem? Please describe.**
I made an executable Unity environment and attempted to training using Python API, while I found the simulation speed of  environment is same as run in unity editor(normal speed). So I'd like to know how to set the training simulation speed(don't use for visual,  faster than normal speed)

**Describe the solution you'd like**
Given some demos or documents to this problem

**Comment by Woojin-unity** [comment]
When you are using Python API, The EngineConfiguration side Channel allows you to modify the time-scale.

See this [Python-API](https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Python-API.md)
![image](https://user-images.githubusercontent.com/97542081/149125423-8fbf47f3-f9d2-4e2b-90c0-c96743f425b3.png)

**Comment by miguelalonsojr** [comment]
Yes, you can set theses settings as follows:

`
    time_scale = 20
    capture_frame_rate = 60
    target_frame_rate = 300
    eng_channel = EngineConfigurationChannel()
    params_channel = EnvironmentParametersChannel()
    env = UnityEnvironment(file_name=file_name, seed=42, side_channels=[eng_channel, params_channel],
                           base_port=5004 + rank, no_graphics=no_graphics)
    engine_config = {"width": 80,
                     "height": 80,
                     "quality_level": 0,
                     "time_scale": time_scale,
                     "capture_frame_rate": capture_frame_rate,
                     "target_frame_rate": target_frame_rate
                     }
    eng_channel.set_configuration_parameters(**engine_config)
`

**Comment by miguelalonsojr** [comment]
Hi Woojin,

I’ve answered your question on the GitHub issue. Feel free to follow up with me directly or through GitHub. We also have an ml-agents channel on slack that you can reach out on: #devs-ml-agents.

Cheers,
Miguel




Miguel Alonso Jr, Ph.D.
(he/him)
Manager, Behavior Authoring and Training
***@***.***
unity.com

**Comment by hxx1996** [comment]
@miguelalonsojr @Woojin-unity 
Thank you very much for your help! I implemented training acceleration in unity executable environment(time-scale=20), but I found another problem:the training speed decreased over time :

![image](https://user-images.githubusercontent.com/38696314/151317706-5ec65460-13b3-4175-9c52-a421c4cee37e.png)
I would like to know what causes this problem and how to solve it. Before that, I also tried to train directly with ML agents. Without packing the environment, the training speed would still decrease.
 
BTW, I found the edit>project settings>time> time scale in the unity editor able to acclerate the simulation speed. Can I speed up both the Unity environment Settings and ML Agents at the same time?  (Theoretically, should get training accelerometers multiplied?)
